lib/experimental/new_embed.dart

```diff
@@ -86,6 +89,8 @@ class NewEmbed {
   ConsoleController consoleExpandController;
   DElement webOutputLabel;
 
+  MDCLinearProgress linearProgress;
```

Comment:
Nice. This'll be way better than my hacked-up animated GIF.

Reply:
👍 - for now it appears at the top of the screen at full width